### ISSUE TITLE
Invoke-AzureRmVMRunCommand doesn't return any output

### ISSUE THREAD

**Issue by alexandair** [Azure/azure-powershell#6302]
### Description

I've tried Invoke-AzureRmVMRunCommand with "Ipconfig" and "RDPSettings" command IDs and those commands didn't return any output. Including the -Debug switch, I've noticed that info about an IP config and RDP settings are there but only in debug stream. They don't go into output stream and that's why we don't see anything.

### Script/Steps for Reproduction

```powershell

PS C:\>
PS C:\> Invoke-AzureRmVMRunCommand -ResourceGroupName aa-lab -VMName aa-labvm -CommandId RDPSettings -Verbose
VERBOSE: Performing the operation "Invoke" on target "aa-labvm".

PS C:\>

```

### Module Version

```powershell

Get-Module -Name AzureRM -ListAvailable
    Directory: C:\Program Files\WindowsPowerShell\Modules


ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Script     6.1.0      AzureRM

```

### Environment Data

```powershell

$PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.14393.2248
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14393.2248
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

```

### Debug Output

```

DEBUG: ============================ HTTP RESPONSE ============================

Status Code:
OK

Headers:
Pragma                        : no-cache
x-ms-ratelimit-remaining-resource:
Microsoft.Compute/GetOperation3Min;14995,Microsoft.Compute/GetOperation30Min;29995
Strict-Transport-Security     : max-age=31536000; includeSubDomains
x-ms-served-by                : ee6753d3-35ce-4152-86fd-a048b0fdfe22_131670835328416047
x-ms-request-id               : 053de2b5-86ba-4882-8634-1be7807e19ca
Cache-Control                 : no-cache
Server                        : Microsoft-HTTPAPI/2.0,Microsoft-HTTPAPI/2.0
x-ms-ratelimit-remaining-subscription-reads: 14998
x-ms-correlation-request-id   : 065434e0-02e3-4240-8431-a32606a25e02
x-ms-routing-request-id       : WESTUS2:20180525T093814Z:065434e0-02e3-4240-8431-a32606a25e02
X-Content-Type-Options        : nosniff
Date                          : Fri, 25 May 2018 09:38:14 GMT

Body:
{
  "startTime": "2018-05-25T11:37:12.8183357+02:00",
  "endTime": "2018-05-25T11:38:02.2404791+02:00",
  "status": "Succeeded",
  "properties": {
    "output": [
      {
        "code": "ComponentStatus/StdOut/succeeded",
        "level": "Info",
        "displayStatus": "Provisioning succeeded",
        "message": "Not domain joined\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal
Server\\Winstations\\RDP-Tcp\\PortNumber: 3389\\nHKLM:\\SOFTWARE\\Policies\\Microsoft\\Windows NT\\Terminal
Services\\fDenyTSConnections: \\nHKLM:\\SOFTWARE\\Policies\\Microsoft\\Windows NT\\Terminal
Services\\KeepAliveEnable: 1\\nHKLM:\\SOFTWARE\\Policies\\Microsoft\\Windows NT\\Terminal
Services\\KeepAliveInterval: 1\\nHKLM:\\SOFTWARE\\Policies\\Microsoft\\Windows NT\\Terminal
Services\\KeepAliveTimeout: 1\\nHKLM:\\SOFTWARE\\Policies\\Microsoft\\Windows NT\\Terminal
Services\\fDisableAutoReconnect: 0\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal
Server\\Winstations\\RDP-Tcp\\fInheritReconnectSame: 1\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal
Server\\Winstations\\RDP-Tcp\\fReconnectSame: 0\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal
Server\\Winstations\\RDP-Tcp\\fInheritMaxSessionTime: 1\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal
 Server\\Winstations\\RDP-Tcp\\fInheritMaxDisconnectionTime:
1\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal Server\\Winstations\\RDP-Tcp\\MaxDisconnectionTime:
0\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal Server\\Winstations\\RDP-Tcp\\MaxConnectionTime:
0\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal Server\\Winstations\\RDP-Tcp\\fInheritMaxIdleTime:
1\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal Server\\Winstations\\RDP-Tcp\\MaxIdleTime:
0\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal Server\\Winstations\\RDP-Tcp\\MaxInstanceCount:
4294967295"
      },
      {
        "code": "ComponentStatus/StdErr/succeeded",
        "level": "Info",
        "displayStatus": "Provisioning succeeded",
        "message": ""
      }
    ]
  },
  "name": "64ca8877-80c6-4085-b65a-6aa978892dc9"
}

DEBUG: ============================ HTTP RESPONSE ============================

Status Code:
OK

Headers:
Pragma                        : no-cache
x-ms-ratelimit-remaining-resource:
Microsoft.Compute/GetOperation3Min;14994,Microsoft.Compute/GetOperation30Min;29994
Strict-Transport-Security     : max-age=31536000; includeSubDomains
x-ms-served-by                : ee6753d3-35ce-4152-86fd-a048b0fdfe22_131670835328416047
x-ms-request-id               : 3f01ddc0-d5ee-4ed9-a551-38c6df645b8d
Cache-Control                 : no-cache
Server                        : Microsoft-HTTPAPI/2.0,Microsoft-HTTPAPI/2.0
x-ms-ratelimit-remaining-subscription-reads: 14997
x-ms-correlation-request-id   : 821d084c-cd62-48cb-b06c-660b501fc681
x-ms-routing-request-id       : WESTUS2:20180525T093814Z:821d084c-cd62-48cb-b06c-660b501fc681
X-Content-Type-Options        : nosniff
Date                          : Fri, 25 May 2018 09:38:14 GMT

Body:
[{"code":"ComponentStatus/StdOut/succeeded","level":"Info","displayStatus":"Provisioning
succeeded","message":"Not domain joined\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal
Server\\Winstations\\RDP-Tcp\\PortNumber: 3389\\nHKLM:\\SOFTWARE\\Policies\\Microsoft\\Windows NT\\Terminal
Services\\fDenyTSConnections: \\nHKLM:\\SOFTWARE\\Policies\\Microsoft\\Windows NT\\Terminal
Services\\KeepAliveEnable: 1\\nHKLM:\\SOFTWARE\\Policies\\Microsoft\\Windows NT\\Terminal
Services\\KeepAliveInterval: 1\\nHKLM:\\SOFTWARE\\Policies\\Microsoft\\Windows NT\\Terminal
Services\\KeepAliveTimeout: 1\\nHKLM:\\SOFTWARE\\Policies\\Microsoft\\Windows NT\\Terminal
Services\\fDisableAutoReconnect: 0\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal
Server\\Winstations\\RDP-Tcp\\fInheritReconnectSame: 1\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal
Server\\Winstations\\RDP-Tcp\\fReconnectSame: 0\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal
Server\\Winstations\\RDP-Tcp\\fInheritMaxSessionTime: 1\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal
 Server\\Winstations\\RDP-Tcp\\fInheritMaxDisconnectionTime:
1\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal Server\\Winstations\\RDP-Tcp\\MaxDisconnectionTime:
0\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal Server\\Winstations\\RDP-Tcp\\MaxConnectionTime:
0\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal Server\\Winstations\\RDP-Tcp\\fInheritMaxIdleTime:
1\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal Server\\Winstations\\RDP-Tcp\\MaxIdleTime:
0\\nHKLM:\\SYSTEM\\CurrentControlSet\\Control\\Terminal Server\\Winstations\\RDP-Tcp\\MaxInstanceCount:
4294967295"},{"code":"ComponentStatus/StdErr/succeeded","level":"Info","displayStatus":"Provisioning
succeeded","message":""}]


DEBUG: AzureQoSEvent: CommandName - Invoke-AzureRmVMRunCommand; IsSuccess - True; Duration -
00:09:05.6087247; Exception - ;


Continue with this operation?
[Y] Yes  [A] Yes to All  [H] Halt Command  [S] Suspend  [?] Help (default is "Y"):



DEBUG: AzureQoSEvent: CommandName - Invoke-AzureRmVMRunCommand; IsSuccess - True; Duration -
00:09:05.6087247; Exception - ;

Confirm
Continue with this operation?
[Y] Yes  [A] Yes to All  [H] Halt Command  [S] Suspend  [?] Help (default is "Y"):
DEBUG: Finish sending metric.

Confirm
Continue with this operation?
[Y] Yes  [A] Yes to All  [H] Halt Command  [S] Suspend  [?] Help (default is "Y"):
DEBUG: 11:46:31 AM - InvokeAzureRmVMRunCommand end processing.

Confirm
Continue with this operation?
[Y] Yes  [A] Yes to All  [H] Halt Command  [S] Suspend  [?] Help (default is "Y"):
DEBUG: 11:46:33 AM - InvokeAzureRmVMRunCommand end processing.

Confirm
Continue with this operation?
[Y] Yes  [A] Yes to All  [H] Halt Command  [S] Suspend  [?] Help (default is "Y"):


PS C:\>

```

**Comment by maddieclayton** [comment]
@ejarvi @hyonholee Can you take a look?

Related (duplicate?) issue: https://github.com/Azure/azure-powershell/issues/5982

**Comment by ejarvi** [comment]
@hyonholee  I'm not currently working in this area, could you take a look?   I could be reading this issue wrong but it seems that both the remote stdout and stderr output (captured in the "message" fields seen in the output above) would be relayed and echoed to the local console by the cmdlet.

**Comment by twitchax** [comment]
Closing in favor of #5982.

**Comment by markcowl** [comment]
actually closing, this was fixed in #6720
